themes/jaeger-docs/layouts/index.html

```diff
@@ -3,12 +3,37 @@
 {{ with resources.GetRemote "https://api.rss2json.com/v1/api.json?rss_url=https://medium.com/feed/jaegertracing" }}
   {{ $json = . | transform.Unmarshal }}
 {{ end }}
-{{ $posts        := first 100 $json.items }}
+{{ $posts := first 100 $json.items }}
+{{ $imgLinks := slice }}
+{{ $defaultImgLink := "/img/jaeger-icon-color.png" }}
+
+{{/* Parse Medium's post first image link */}}
+{{ range $k, $v := $posts }}
+  {{ $imgLink := "" }}
+  {{ with strings.FindRESubmatch "<img[^>]+src=\"([^\">]+)\"" (index $v "content") 1 }}
+    {{ if not . }} {{/* If no image found in post */}}
+      {{ $imgLink = $defaultImgLink }}
```

Comment:
you don't have to keep assigning default img, just assign it once in L12 and then override only when you find something more useful. This will make the code simpler.